### PR TITLE
Now You See Me: Remove overloaded word from GLabs containers

### PR DESCRIPTION
**Pull request by katebee**
## What does this change?

Switches labs containers to use classnames that would have zero chance of appearing on an adblock list, or elsewhere in our codebase, ever. The choice of `adverts` for labs content is unfortunate since it is too generic, and is also used for our actual adverts.

- Remove overloaded word `adverts` from usage in Labs containers
- Opinionated CSS overhaul 
- Fixes some layout bugs that were discovered during testing

See also: #16567 #16101

## What is the value of this and can you measure success?

- Labs content is visible to more people
- Slightly less CSS, and slightly flatter CSS
- Consolidation of some class styling so all the related CSS is in one place
- Moving commercial CSS closer to [the frontend css conventions](https://github.com/guardian/frontend/blob/master/docs/99-archives/css-guidelines.md)
- Reduces risk of classname collisions

## Does this affect other platforms - Amp, Apps, etc?

Nope - only Desktop

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

Oh yes 😎 

##### Before, with uBlock Origin / Opera AB enabled / Firefox & ABP :

picture 932

##### After, with uBlock Origin:

picture 933

##### After, with Opera AB enabled:

picture 933

##### After, with Firefox ABP:

picture 935

## Bonus! Layout fixes! ✨💅

- Tablet breakpoint range was doing something weird with container titles
- Contents of PaidFor container did not centre align at tablet and below breakpoints
- PaidFor `more` buttons had margin issues at tablet and below breakpoints

This PR ensures the Labs containers follow the conventions for editorial containers.

##### Before:

picture 930

picture 927

##### After:

picture 931

picture 928

## Tested in CODE?

Nope, but extensive testing during development cycle at various breakpoints
Brings the containers back in Opera AB mode, and Firefox ABP too 😎

### CODE REVIEW

**Comment by thiagodebastos** [comment]
@katebee Great! I'm excited to see this and optimistic about the resulting viewership increase 👍

File: commercial/app/views/contentapi/item.scala.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
 }{
 
-    <div class="adverts__row adverts__row--single">
+    <div class="dumathoin__row dumathoin__row--single">
```
Comment (regiskuckaertz) [inline commercial/app/views/contentapi/item.scala.html]: 🐱  => 🐭 

😂

Comment (katebee) [inline commercial/app/views/contentapi/item.scala.html]: 💯 🐱 🐭

:rofl:

**Comment by regiskuckaertz** [comment]
What about `.advert`?

**Comment by katebee** [comment]
@regiskuckaertz I started changing classnames and stopped when I achieved the desired result. However, I did finish off changing all the containerWrapper classes because I like a sense of completeness.

`.advert` is a misnomer in the context of labs (and one of the reasons I decided to rename the files) but does not get blocked at this time. The work may be worth doing if it does get targeted in the future, or if we want our CSS to be more understandable.

**Comment by regiskuckaertz** [comment]
Fair enough 👍

File: commercial/app/views/contentapi/item.scala.html (hunk 1 above)
Comment (thiagodebastos) [inline commercial/app/views/contentapi/item.scala.html]: I'm down for `.clanggedin-silverbeard` any day of the week.

**Pull request merged**